webpush_channels/views/channels.py

```diff
@@ -72,8 +72,7 @@ def retrieve_channel_information(request):
     user_registered = [r for r in registrations if r['id'] == request.prefixed_userid]
 
     if not user_registered:
-        request.response.status = 403
-        return {}
+        raise httpexceptions.HTTPForbidden()
```

Comment:
It is because we have a test that proves it: https://github.com/webpush-channels/webpush-channels/pull/31/files#diff-c4810312bfbd1af5b4bb860290d82c97R198